web/modules/custom/maybe_forms/maybe_forms.module

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Custom Blocks module.
+ */
+
+const MAYBE_FORMS_FARMER_ROLE = 'farmer';
+const MAYBE_FORMS_CONSUMER_ROLE = 'consumer';
+const MAYBE_FORMS_UNIQUE_ERROR_MESSAGE = "Your email has been used.";
+const MAYBE_FORMS_ERROR_MESSAGE = "An error occurred while creating the farm.";
+const MAYBE_FORMS_STORE_TYPE = "online";
+const MAYBE_FORMS_DEFAULT_PASSWORD = "123456";
```

Comment:
Password should be an encoded string of smth (for example: the timestamp when user is created)

Reply:
In the feature there is no feature to send the user the password, setting the default password is a temporary function in this feature.